resources/tags.yaml

```diff
@@ -0,0 +1,111 @@
+accounting:
+  title: 'Accounting'
+  description: 'Processing and communication of financial information'
+analytics:
+  title: 'Analytics'
+  description: 'Data or statistics to make intelligent marketing and business decisions'
+backend: 
+  title: 'Backend'
+  description: 'Backend'
+calendars:
+  title: 'Calendars'
+  description: 'Calendars'
+cloud_platform: 
+  title: 'Cloud Platform'
+  description: 'Offers hosting on the same supporting infrastructure'
+collaboration: 
+  title: 'Collaboration'
+  description: 'Collaboration'
+crm: 
+  title: 'CRM'
+  description: 'Customer Relationship Management'  
+customer_support:
+  title: 'Customer Support'
+  description: 'Range of customer services to assist users of a product'
+dashboard: 
+  title: 'Dashboard'
+  description: 'Dashboard'
+developer_tools: 
+  title: 'Developer Tools'
+  description: 'Developer Tools'
+ecommerce:
+  title: 'eCommerce'
+  description: 'Facilitation of trading in products or services using computer networks'  
+education:
+  title: 'Education'
+  description: 'Education'
+email: 
+  title: 'Email'
+  description: 'Email'
+enterprise: 
+  title: 'Enterprise'
+  description: 'Enterprise'
+event_management:
+  title: 'Event Management'
+  description: 'Event Management'
+financial:
+  title: 'Financial'
+  description: 'Financial'
+frontend:
+  title: 'Frontend'
+  description: 'Frontend'
+forms: 
+  title: 'Forms'
+  description: 'Forms'
+hosting: 
+  title: 'Hosting'
+  description: 'Hosting'
+iot:
+  title: 'IoT'
+  description: 'Internet of Things'
+location:
+  title: 'Location'
+  description: 'Location'
+machine_learning:
+  title: 'Machine Learning'
+  description: 'Pattern recognition and computational learning theory'
+marketing: 
+  title: 'Marketing'
+  description: 'Marketing'
+media:
+  title: 'Media'
+  description: 'Media content'
+messaging:
+  title: 'Messaging'
```

Comment:
Hello @MikeRalphson , I added the description, but continues the comment visible, if possible remove this comment. The problem is that the reference was to ' title ' instead of the line ' description' . Thank you